custom_model_runner/README.md

```diff
@@ -308,7 +308,7 @@ adding the training model into DataRobot.
 You can try this out on our sklearn classifier model template this this command:
 
 ```
-drum fit --code-dir task_templates/pipelines/python3_sklearn_binary --target-type binary --target Species --input \
+drum fit --code-dir task_templates/3_pipelines/python3_sklearn_binary --target-type binary --target Species --input \
```

Comment:
A little bit context included in the PR description would be helpful. What is this renaming?

Reply:
Sorry, that Rationale and Summary was not very helpful. I added a much longer summary & rationale above